modules/calibration/data/dev_kit/perception_launch/dev_kit_perception_lidar.launch

```diff
@@ -9,10 +9,10 @@
          ./conf/dag_streaming_0.conf -->
 
     <module>
-        <name>perception</name>
+        <name>perception_lidar</name>
```

Comment:
I think the module name 'perception' is right, 'perception_lidar' is not a module. If there is not any other wrong for the module name, please keep it with 'perception'.

Reply:
The camera launch name is `perception_camera`, [Link](https://github.com/ApolloAuto/apollo/blob/master/modules/calibration/data/dev_kit/perception_launch/dev_kit_perception_camera.launch)

I just want to be consistent with the `dev_kit_perception_camera.launch`. This name is just a print prompt and has no other meaning. If it is convenient for users to distinguish, I suggest changing to a more detailed name.

```
        <name>perception_camera</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_streaming_perception_dev_kit_camera.dag</dag_conf>
        <!-- if not set, use default process -->
        <process_name></process_name>
```